src/Microsoft.Diagnostics.Tracing.Eventing/EventPipeClient.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>Microsoft.Diagnostics.Tracing.Eventing</RootNamespace>
+    <IsPackable>False</IsPackable>
+    <PackAsTool>False</PackAsTool>
```

Comment:
This is only needed for global CLI tools.  All you need is IsPackable and maybe these:

    <Description>Diagnostic trace support</Description>
    <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
    <PackageTags>trace</PackageTags>

Reply:
`<TargetFramework>netcoreapp2.1</TargetFramework>` needed because `Socket` is not in `netstandard2.0`
`<RootNamespace>Microsoft.Diagnostics.Tracing.Eventing</RootNamespace>` to tell VS to add new classes under this namespace
Thought `<IsPackable>False</IsPackable>` was needed to ship this library (later) out of band.

Reply:
I was just talking about <PackAsTool> but I didn't see that it was false, but you don't need it ever (for only global tools).

Setting <IsPackable> to false is fine to doc that you don't want packages built yet.

RootNamespace is fine.

I didn't even notice that TargetFramework was netcoreapp2.1 and would have said something. Being netcoreapp2.1 it might cause building dotnet-trace (I assume it references it) as a global tool a problem. I vaguely remember some problem with some other assembly that was including by a dotnet global tool.